VideoWeb/VideoWeb/Controllers/ParticipantsController.cs

```diff
@@ -133,26 +155,14 @@ public async Task<IActionResult> UpdateParticipantDisplayNameAsync(Guid conferen
             return NoContent();
         }
 
-        private async Task UpdateCacheAndPublishUpdate(Guid conferenceId)
-        {
-            var conference = await conferenceService.ForceGetConference(conferenceId);
-            var mappedParticipants = conference.Participants.Select(ParticipantDtoForResponseMapper.Map).ToList();
-            await eventHandlerFactory.Get(EventHub.Enums.EventType.ParticipantsUpdated).HandleAsync(new CallbackEvent
-            {
-                Participants = mappedParticipants,
-                ParticipantsToNotify = mappedParticipants,
-                ConferenceId = conferenceId,
-                EventType = EventHub.Enums.EventType.ParticipantsUpdated,
-                Reason = "Participant display name updated",
-                TimeStampUtc = DateTime.UtcNow
-            });
-        }
-
         /// <summary>
-        /// Get the participant details of a conference by id for VH officer
+        /// Retrieves participants with contact details for a conference by conference
```

Comment:
The original summary was fine here, no need to change it

```
/// <summary>
/// Get the participant details of a conference by id for VH officer
/// </summary>
/// <param name="conferenceId">The unique id of the conference</param>
/// <returns>the participant details, if permitted</returns>
```

Reply:
done